src/components/Captures/CaptureTable.js

```diff
@@ -120,7 +121,7 @@ const CaptureTable = () => {
   useEffect(async () => {
     // Don't do anything if there are no captures
     if (!captures?.length) {
-      return;
+      return isLoading;
```

Comment:
Nothing uses the return value from this function, so this doesn't do anything.
The `isLoading` flag should be set to `true` before the API is called and `false` when the response is received. See  VerifyContext for guidance:
https://github.com/Greenstand/treetracker-admin-client/blob/4498a21d77b7c02ab1647248df303a603a5da198/src/context/VerifyContext.js#L142

This flag should then be used in CapturesTable to determine whether to show the loading indicator.